To: vim_dev@googlegroups.com
Subject: Patch 7.3.370
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.370
Problem:    Compiler warns for unused variable in Lua interface.
Solution:   Remove the variable.
Files:	    src/if_lua.c


*** ../vim-7.3.369/src/if_lua.c	2011-09-21 17:15:34.000000000 +0200
--- src/if_lua.c	2011-12-01 20:19:17.000000000 +0100
***************
*** 1044,1056 ****
      static int
  luaV_open(lua_State *L)
  {
-     luaV_Buffer *b;
      char_u *s = NULL;
  #ifdef HAVE_SANDBOX
      luaV_checksandbox(L);
  #endif
      if (lua_isstring(L, 1)) s = (char_u *) lua_tostring(L, 1);
!     b = luaV_pushbuffer(L, buflist_new(s, NULL, 1L, BLN_LISTED));
      return 1;
  }
  
--- 1044,1055 ----
      static int
  luaV_open(lua_State *L)
  {
      char_u *s = NULL;
  #ifdef HAVE_SANDBOX
      luaV_checksandbox(L);
  #endif
      if (lua_isstring(L, 1)) s = (char_u *) lua_tostring(L, 1);
!     luaV_pushbuffer(L, buflist_new(s, NULL, 1L, BLN_LISTED));
      return 1;
  }
  
*** ../vim-7.3.369/src/version.c	2011-12-08 15:57:54.000000000 +0100
--- src/version.c	2011-12-08 15:59:35.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     370,
  /**/

-- 
Emacs is a nice OS - but it lacks a good text editor.
That's why I am using Vim.  --Anonymous

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
